app/controllers/concerns/form_attachment_create.rb

```diff
@@ -2,11 +2,29 @@
 
 module FormAttachmentCreate
   extend ActiveSupport::Concern
+  include SentryLogging
 
   def create
-    validate_file_upload_class!
-    save_attachment_to_cloud!
-    save_attachment_to_db!
+    begin
```

Comment:
Would these begin --> rescue blcoks be better within the methods that are being called? adding them all here makes this a lot hard to read in my opinion.

Reply:
👍 Seems like that'd be a good move to keep the main method clear while still being plenty easy to clean up unnecessary stuff once this is understood + fixed.